deb-x64/Dockerfile

```diff
@@ -156,15 +157,10 @@ RUN curl -Sl -O https://dd-agent-omnibus.s3.amazonaws.com/kernel-4.9-headers-deb
     && rm kernel-4.9-headers-deb-x64.tgz
 
 # Go
-# Starting with go1.16, we need to build go from source, as the publicly distributed
-# go binaries are built with a linker (binutils) version that's higher than what we have in
-# the deb_x64 builder (the builder has 2.22, while go1.16 uses 2.26). Therefore, we need to
-# build go manually in the builder for it to work with the linker present in the builder image.
-# See: https://github.com/golang/go/issues/43996#issuecomment-782547917
-# TODO: investigate if we could use gvm to do this for us.
-COPY setup_go.sh /
-RUN ./setup_go.sh
```

Comment:
Do we still need bison environment variables in the `Dockerfile` now that this script isn't executed?

I think `binutils` is still required when building gcc though

Reply:
Right, I'll remove them, first checking if it works (retriggering the CI)